enzy_htp/structure/residue.py

```diff
@@ -233,6 +233,11 @@ def n_side_residue(self):
                                 " Something wrong in your residue indexing. "
                                 "It could be your index is not continous")
         return result
+    
+    def hydrogens(self):
```

Comment:
Make it a @property, it is a convention for getting strutcure components such as Structure().residues
(remember to change all the reference too i.e.: remove the "()")

Reply:
Resolved.